docs/how-to/code/build-docs/task.yaml

```diff
@@ -0,0 +1,39 @@
+###########################################
+# IMPORTANT
+# Comments matter!
+# The docs use the wrapping comments as 
+# markers for including said instructions 
+# as snippets in the docs.
+###########################################
+summary: test the "Build the docs" guide
+
+prepare: |
+  tests.pkgs install python3-venv python3-dev libapt-pkg-dev
+
+execute: |
+  pushd $PROJECT_PATH/
+
+  # [docs:create-env]
+  python3 -m venv docs/env
+  source docs/env/bin/activate
+  # [docs:create-env-end]
+
+  # [docs:install-deps]
+  pip install -r requirements-focal.txt -r requirements-doc.txt
+  # [docs:install-deps-end]
+    
+  # [docs:make-docs]
+  make docs  # the home page can be found at docs/_build/html/index.html
+  # [docs:make-docs-end]
+
+  set +e
+  timeout -s SIGINT 7 \
+  make rundocs
+  # [docs:make-rundocs-end]
+  ret=$?
+  set -e
+
+  if [ $ret -ne 124 ]
+  then
+    exit $ret
+  fi
```

Comment:
can you please add a newline at the end of these files? Many of them are missing it and github complains